compose/container.py

```diff
@@ -14,6 +14,7 @@ def __init__(self, client, dictionary, has_been_inspected=False):
         self.client = client
         self.dictionary = dictionary
         self.has_been_inspected = has_been_inspected
+        self._version = '1.17'
```

Comment:
Reading the docker-py clinet.py code, it may be appear your correct.  I did actually test this in a real world environment, and I thought everything was necessary at the time, but I was doing the job at 3am, so its possible I made an error.

iI'll test without it next week and either submit results or submit a patch revision with this change reverted.